iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -90,7 +90,6 @@ private InputFormatConfig() {
   public static final String QUERY_FILTERS = "iceberg.query.filters";
 
   public enum InMemoryDataModel {
-    PIG,
```

Comment:
@maswin, is it ok to remove? any side effects on https://github.com/apache/hive/pull/4446?

Reply:
Thanks for checking, @maswin!